index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const { getAllMovies, getMoviesByDirectors, getMovieByTitle, saveNewMovie } = require('./controllers/movies')
+
+const app = express()
+
+app.get('/', getAllMovies)
+
+app.get('/:director', getMoviesByDirectors)
+
+app.get('/:title', getMovieByTitle)
```

Comment:
This is the same route twice. When the user types `/lord` into the URL the code has no way of knowing if that is a director or a title. Regardless of what the user types they will get the first route (line 9) and the second (line 11) is unreachable.